src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdServerConfiguration.cs

```diff
@@ -123,6 +123,11 @@ public void Configure(string name, OpenIddictServerOptions options)
                 options.AccessTokenHandler = new JwtSecurityTokenHandler();
             }
 
+            if (!settings.EnableAuthorizationEndpoint && !settings.EnableTokenEndpoint && !settings.EnableLogoutEndpoint && !settings.EnableUserInfoEndpoint)
+            {
+                // Work around for Openiddict that throws an exception when no flows are selected
+                settings.EnableAuthorizationEndpoint = true;
```

Comment:
Yeah no, we can't do that (enabling disabled endpoints is just evil).
I'm not even sure it fully fixes the issue as you'll probably get an exception indicating no flow was configured, even if you enable the authorization endpoint.

Reply:
Going to remove this, since this is no longer neccesairy